src/main/java/com/sjsu/backbenchers/vBless/Constants.java

```diff
@@ -0,0 +1,14 @@
+package com.sjsu.backbenchers.vBless;
+
+public interface Constants {
```

Comment:
Can you please create a class and static final variables for constants. Merging for now, pls update next time.